tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -278,7 +278,7 @@ describe('amo/components/AddonReviewList', () => {
         .toHaveText(fakeReview.title);
 
       expect(root.find('.AddonReviewList-li p'))
-        .toHaveText(fakeReview.body);
+        .toHaveHTML(`<p>${fakeReview.body}</p>`);
```

Comment:
This isn't actually checking for a `<br />` tag, which is the point of this patch. This test should actually revert to `.toHaveText(fakeReview.body);` I would think (it should still pass).